src/components/CoreApps/index.js

```diff
@@ -0,0 +1,91 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import React from 'react'
+import getLatestVersion from '../../get-latest-version'
+import AppList from '../AppList'
+
+const coreAppNames = [
+    'App Management',
+    'Cache Cleaner',
+    'Capture',
+    'Dashboard',
+    'Data Visualizer',
+    'Datastore Management',
+    'Import/Export',
+    'Interpretations',
+    'Maintenance',
+    'Menu Management',
+    'Messaging',
+    'Reports',
+    'SMS Configuration',
+    'Scheduler',
+    'Settings',
+    'Translations',
+    'User Management',
+]
+
+const query = {
+    coreApps: {
+        resource: 'apps',
+        params: {
+            bundled: true,
+        },
+    },
+    appHub: {
+        resource: 'appHub/v1/apps',
+    },
+}
+
+const CoreApps = () => {
+    const { loading, error, data } = useDataQuery(query)
+
+    const overridenCoreApps = data?.coreApps.filter(app => app.bundled)
+    const apps = coreAppNames
+        .map(coreAppName => {
+            const overridenApp = overridenCoreApps?.find(
+                a => a.name === coreAppName
+            )
+            if (overridenApp) {
+                return overridenApp
+            }
+            return {
+                name: coreAppName,
+                bundled: true,
+                short_name: coreAppName,
+                developer: {
+                    company: 'DHIS2',
+                },
+                icons: {},
+            }
+        })
+        .map(app => ({
+            ...app,
+            appHub: data?.appHub.find(({ id, name, developer }) => {
+                if (app.app_hub_id) {
+                    return id === app.app_hub_id
+                }
+                return name === app.name && developer.organisation === 'DHIS2'
```

Comment:
I think we should probably enforce *only* matching by app hub ID.  That will require us to set the app hub id for all core apps before releasing 2.36 which is a very small, low-risk change but also needs all apps on the app hub (which we should probably do in any case).

What do you think?  Leaving the "soft fallback" for bundled apps does give us more time to get the infrastructure set up...

If we keep this "fallback" it should only be supported for apps where `isBundled === true`, I think.

Reply:
Yep I agree let's do only matching by App Hub ID. I don't see why we wouldn't be able to set the App Hub ID for all core apps before 2.36.